test/v2/live/live_network_test.go

```diff
@@ -298,28 +262,23 @@ func smallBlobDispersalAllQuorumsSetsTest(t *testing.T, environment string) {
 	rand := random.NewTestRandom()
 	payload := rand.VariableBytes(units.KiB, 2*units.KiB)
 
-	config, err := client.GetConfig(environment)
-	require.NoError(t, err)
-
 	c := client.GetTestClient(t, environment)
 
 	t.Run("0 1", func(t *testing.T) {
-		checkAndSetCertVerifierAddress(t, c, config.EigenDACertVerifierAddressQuorums0_1)
-		err = testBasicDispersal(t, c, payload)
-		require.NoError(t, err)
-	})
-
-	t.Run("0 1 2", func(t *testing.T) {
-		checkAndSetCertVerifierAddress(t, c, config.EigenDACertVerifierAddressQuorums0_1_2)
-		err = testBasicDispersal(t, c, payload)
+		err := testBasicDispersal(t, c, payload)
 		require.NoError(t, err)
 	})
 
-	t.Run("2", func(t *testing.T) {
-		checkAndSetCertVerifierAddress(t, c, config.EigenDACertVerifierAddressQuorums2)
-		err = testBasicDispersal(t, c, payload)
-		require.NoError(t, err)
-	})
+	// We need to eventually re-enable testing with quorum set {0,1,2} and {2}
```

Comment:
Optionally, you could keep these tests, and make them simply use the raw `CertVerifier` addresses, just like they were doing before they got disabled. Presence of the `CertVerifierRouter` doesn't prevent usage of the `CertVerifier` directly, thought this would complicate the test logic.

Up to you